native/packages/react-native-bpk-component-badge/stories.js

```diff
@@ -21,11 +21,13 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react-native';
 import { View, StyleSheet } from 'react-native';
+import { icons } from 'react-native-bpk-component-icon';
 import {
   colorGray50,
   colorGray700,
   spacingSm,
 } from 'bpk-tokens/tokens/base.react.native';
+import { capitalize } from 'lodash';
```

Comment:
Why are we adding an import instead of doing it in a one-liner that we already had?

Reply:
Why write a one-liner that we don't already have when we could just import it from a library we do already have 😆

Reply:
Counter-counter question/statement: You **don't** already have Lodash; it isn't in this package's dependencies 😉

Reply:
Counter-counter-counter statement: Yeah but this is stories 🤷‍♂️ 
So we do have Lodash

Reply:
Counter-counter-counter-counter statement: It's only working because you have Lodash installed elsewhere - by that logic why do we add anything at all to `devDependencies` 😛

Reply:
So what d'yall suggest then?